backend/main.py

```diff
@@ -20,51 +19,4 @@
   allow_headers=["*"],
 )
 
-class BobaOptionMilkType(graphene.ObjectType):
-  name = graphene.String()
-
-class BobaOptionToppingsType(graphene.ObjectType):
-  name = graphene.String()
-  price = graphene.Float()
-
-class BobaOptionType(graphene.ObjectType):
-  milk = graphene.Field(lambda: BobaOptionMilkType)
-  toppings = graphene.List(lambda: BobaOptionToppingsType)
-
-
-class BobaType(graphene.ObjectType):
-  id = graphene.ID()
-  name = graphene.String()
-  description = graphene.String()
-  price = graphene.Float()
-  imageLink = graphene.String()
-  options = graphene.Field(lambda: BobaOptionType)
-
-class BobaQuery(graphene.ObjectType):
-  bobaList = graphene.List(BobaType)
-
-  def resolve_bobaList(self, info):
-    with open('./database/bobalist.json') as f:
-        bobajson = json.load(f)['data']
-    return bobajson
-
-@app.get('/boba_list')
-async def get_boba_list():
-  with open('./database/bobalist.json') as f:
-    bobaList = json.load(f)
-  return bobaList["data"]
-
-@app.get('/milk_list')
-async def get_milk_list():
-  with open('./database/milkoption.json') as f:
-    milkOption = json.load(f)
-  return milkOption["data"]
-
-@app.get('/toppings_list')
-async def get_toppings_list():
-  with open('./database/toppingsoption.json') as f:
-    toppingsOption = json.load(f)
-  return toppingsOption["data"]

```

Comment:
Does front end still works since you entirely removed toppings and milk endpoints?
Isn't this ticket scoped only for bobalist?

Reply:
updated